hirola/front/forms/user_forms.py

```diff
@@ -422,3 +423,39 @@ class PhoneProfileUserDataCollectionForm(forms.Form):
 
     CHOICES = (('0', 'Quantity',),)
     quantity = forms.CharField(error_messages=quantity_error_messages)
+
+
+class ContactUsForm(forms.Form):
+
+    error_messages = {
+        'invalid': _('Please enter a valid comment'),
+        'required': _("Comment cannot be empty!")
+    }
+
+    name = forms.CharField(required=False, widget=forms.TextInput(
+        attrs={'placeholder': 'Name(Optional)'}))
+    email = forms.EmailField(widget=forms.TextInput(
+        attrs={'placeholder': 'Email'}))
+    comment = forms.CharField(widget=forms.Textarea(
+        attrs={'placeholder': 'Tell us about the issue..',
+               'class': 'materialize-textarea'}))
+
+    def clean_comment(self):
+        comment = self.cleaned_data.get('comment')
+        if not comment or comment.isspace():
+            raise ValidationError(self.error_messages['required'])
+        if re.match(r'^[_\W]+$', comment):
+            raise ValidationError(self.error_messages['invalid'])
+        return comment
+
+    def send_email(self):
+        to_email = 'testermail717@gmail.com'
```

Comment:
The to_email should be captured from the environment variables or values in settings.py. Maybe using https://docs.djangoproject.com/en/2.1/ref/settings/#email-host-user or possibly a new environment vairable added to the settings.py. To get the variable you can use `from django.conf import settings` this is already used somewhere in the code.